lib/mapObjects/CommonConstructors.h

```diff
@@ -30,21 +30,7 @@ class CDefaultObjectTypeHandler : public AObjectTypeHandler
 	{
 		auto obj = new ObjectType();
 		preInitObject(obj);
-
-		if (tmpl)
```

Comment:
We still can test shared_ptr for being empty. Or is such case not possible?

Reply:
that's possible but valid

Reply:
Okay, so now epty pointer will be valid and default, but can be set explicitely. This is needed in some cases.